boards/arm/nrf5340_audio_dk_nrf5340/doc/index.rst

```diff
@@ -0,0 +1,109 @@
+.. _nrf5340_audio_dk_nrf5340:
+
+nRF5340 Audio DK
+################
+
+Overview
+********
+
+The nRF5340 Audio DK (PCA10121) is designed for showcasing, developing and experimenting
+with Bluetooth® LE Audio.
+
+You can use this board for developing LE-Audio-compatible applications that support Auracast™,
+connected isochronous streams (CIS) and broadcast isochronous streams (BIS),
+and offer support for acting as a audio source, audio sink and source + sink.
+
+Zephyr uses the nrf5340_audio_dk_nrf5340 board configuration for building
+for the nRF5340 Audio DK.
+
+Hardware
+********
+
+The nRF5340 Audio DK comes with the following hardware features:
+
+* nRF5340 dual-core SoC based on the Arm® Cortex®-M33 architecture
+* CS47L63 Low-Power Audio DSP with mono differential headphone driver
+* nPM1100 Ultra-small form-factor Power Management IC
+* On-board digital microphone
+* On-board power measurement
+* SD card slot
+* Built-in debugger
+* Stereo analog input using 3.5 mm jack
+* USB soundcard capability
+
+.. figure:: img/nrf5340_audio_dk.jpg
+     :align: center
+     :alt: nRF5340 DK
+
+More information about the board can be found at the `nRF5340 Audio DK website`_.
```

Comment:
The hardware for the Audio DK has been published on Infocenter, so we need to rephrase this part:
```suggestion
More information about the board can be found at the `nRF5340 Audio DK website`_. The `Nordic Semiconductor Infocenter`_
contains the processor's information and the datasheet.
```

Reply:
Fixed